bindings/web5_uniffi/src/web5.udl

```diff
@@ -1,7 +1,6 @@
 namespace web5 {
   JwkData ed25519_generator_generate();
 
-  [Throws=RustCoreError]
```

Comment:
did this mean to get removed?

Reply:
yeah, because we're moving to all `resolve()` functions returning a `ResolutionResult` wherein the error case is represented via the `ResolutionMetadataError`